examples/package.json

```diff
@@ -5,22 +5,25 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "build": "yarn build:clean-state && yarn build:counter && yarn build:counter-lowlevel && yarn build:cross-contract-call && yarn build:fungible-token-lockable && yarn build:fungible-token && yarn build:non-fungible-token && yarn build:status-message-collections && yarn build:status-message",
+    "build": "yarn build:clean-state && yarn build:counter && yarn build:counter-lowlevel && yarn build:counter-ts && yarn build:cross-contract-call && yarn build:fungible-token-lockable && yarn build:fungible-token && yarn build:non-fungible-token && yarn build:status-message-collections && yarn build:status-message",
     "rebuild": "cd .. && yarn build && cd examples && rm -rf node_modules && rm -rf build && yarn && yarn build",
     "build:status-message": "yarn near-sdk build src/status-message.js build/status-message.wasm",
     "build:clean-state": "near-sdk build src/clean-state.js build/clean-state.wasm",
     "build:counter": "near-sdk build src/counter.js build/counter.wasm",
     "build:counter-lowlevel": "near-sdk build src/counter-lowlevel.js build/counter-lowlevel.wasm",
+    "build:counter-ts": "near-sdk build src/counter.ts build/counter-ts.wasm",
```

Comment:
Looks like it is not added to CI (as well as the low-level counter example).

Reply:
Fixed